App.tsx

```diff
@@ -42,35 +44,36 @@ const App: React.FC<Props> = () => {
         }
         await languageDefault();
         dispatch(setLoadingScreen(false));
-        // Important: Do not add any other code here, this is for the root component only.
-        // Add code in the `atAppInit` function below if you need to run any code at app start/restart.
         await atAppInit();
       } catch (error) {
-        console.error('Error fetching initial data:', error);
-      }
+        show(locale.t('ErrorFetchingInitialData'), 'error');
+        console.error(locale.t('ErrorFetchingInitialData'), error);      }
     };
 
     const checkPermission = async () => {
       try {
         await Camera.requestCameraPermission();
       } catch (error) {
-        console.error('Error checking camera permissions:', error);
+        show(locale.t('ErrorCheckingCameraPermissions'), 'error');
+        console.error(locale.t('ErrorCheckingCameraPermissions'), error);
       }
     };
 
     const handleSharedFiles = () => {
       try {
         ReceiveSharingIntent.getReceivedFiles(
-          (files: any) => {
-            console.log('Shared files received:', files);
+          (sharedFiles: any) => {
+            console.log('Shared files received:', sharedFiles);
           },
           (error: any) => {
-            console.error('Error receiving shared files:', error);
+            show(locale.t('ErrorReceivingSharedFiles'), 'error');
+            console.error(locale.t('ErrorReceivingSharedFiles'), error);
           },
-          'etendo'
+          References.EtendoReceiveShare,
         );
       } catch (error) {
-        console.error('Error handling shared files:', error);
+        show(locale.t('ErrorHandlingSharedFiles'), 'error');
+        console.error(locale.t('ErrorHandlingSharedFiles'), error);
```

Comment:
About this changes, my doubt is, is really needed to translate console mesage errors?
I understand the usage of locale.t(error) for the user visible alerts and popups, but, as a developer, I would like to have the original error message in the console